pyangbind/lib/yangtypes.py

```diff
@@ -377,7 +377,7 @@ class TypedList(collections.MutableSequence):
     _list = list()
 
     def __init__(self, *args, **kwargs):
-      self._unique = kwargs.pop('unique', False)
+      self._unique = kwargs.pop('unique', True)
```

Comment:
Per the comments in Slack. I'm a little torn on this one.

If we make the default here `True`, then we'll end up saying that if existing consumers just upgrade `pyangbind` module from PyPi, but don't regenerate bindings, then existing code that expects this to be the previous state will end up being broken.

Sorry for the hassle, but I think requiring regenerating bindings such that the feature is turned on as you'd had this originally is safer to allow consumers to choose when to enable this.